packages/analytics-client-common/src/session.ts

```diff
@@ -0,0 +1,7 @@
+export const isNewSession = (sessionTimeout: number, lastEventTime?: number): boolean => {
+  const _currentTime = Date.now();
+  const _lastEventTime = lastEventTime || Date.now();
+  const timeSinceLastEvent = _currentTime - _lastEventTime;
```

Comment:
the duplicated names make it harder to read so suggesting this

```suggestion
export const isNewSession = (sessionTimeout: number, lastEventTime: number = Date.now()): boolean => {
  const currentTime = Date.now();
  const timeSinceLastEvent = currentTime - lastEventTime;
```